docs/source/deploying-cloud.rst

```diff
@@ -14,9 +14,9 @@ You may want to consider the following options:
     and `Dask-Yarn <https://yarn.dask.org>`_.
 
     Specific documentation for the popular Amazon EMR service can be found
-    `here <https://yarn.dask.org/en/latest/aws-emr.html>`_
+    `here <https://yarn.dask.org/en/latest/aws-emr.html>`_.
 3.  Directly launching cloud resources such as VMs or containers via a cluster manager with
-    `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_
+    `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_ or `Coiled <https://www.coiled.io/>`_.
```

Comment:
From a users perspective I think that these two are pretty different.  It's true that under the hood they both use VMs, but I think that commercial or managed services might make more sense as a separate bullet point.

Reply:
I came here to see if we'd added Coiled to this list. Agreed with Matt that Coiled should be a separate bullet, but I think it definitely makes sense to have on the list.

```suggestion
    `Dask Cloud Provider <https://cloudprovider.dask.org/en/latest/>`_.
4. `Coiled <https://www.coiled.io/>`_, a commercial service that manages launching Dask clusters on your own cloud infrastructure.
```